src/ssr-template-helper.js

```diff
@@ -87,10 +84,7 @@ export class SsrTemplateHelper {
     if (!opt_templates) {
       const template = this.templates_.maybeFindTemplate(element);
       if (template) {
-        // The document fragment can't be used in the message channel API thus
-        // serializeToString for a string representation of the dom tree.
-        mustacheTemplate = this.xmls_.serializeToString(
-            this.templates_.findTemplate(element));
+        mustacheTemplate = template./*REVIEW*/innerHTML;
```

Comment:
Can we clarify in the PR description why we need to use `innerHTML` instead of the previous serialization code?

Reply:
Done.